src/graphql/client.ts

```diff
@@ -15,7 +15,7 @@ const authMiddleware = new ApolloLink((operation, forward) => {
   if (token) {
     operation.setContext({
       headers: {
-        Authorization: `Bearer ${token}`,
+        authorization: `Bearer ${token}`,
         'Accept-Language': i18n.languages[0],
```

Comment:
Should all headers, not just `authorization`, now be lower cased? For consistency.

Reply:
Yes, I changed that one too.